contracts/src/v0.8/l2ep/dev/arbitrum/ArbitrumValidator.sol

```diff
@@ -249,14 +249,14 @@ contract ArbitrumValidator is Validator {
   }
 
   /// @notice internal method that stores the gas configuration
-  function _setGasConfig(uint256 maxGas, uint256 gasPriceBid, uint256 baseFee, address gasPriceL1FeedAddr) internal {
+  function _setGasConfig(uint96 maxGas, uint256 gasPriceBid, uint256 baseFee, address gasPriceL1FeedAddr) internal {
     // solhint-disable-next-line custom-errors
     require(maxGas > 0, "Max gas is zero");
     // solhint-disable-next-line custom-errors
     require(gasPriceBid > 0, "Gas price bid is zero");
     // solhint-disable-next-line custom-errors
     require(gasPriceL1FeedAddr != address(0), "Gas price Aggregator is zero address");
-    s_gasConfig = GasConfig(maxGas, gasPriceBid, baseFee, gasPriceL1FeedAddr);
+    s_gasConfig = GasConfig(gasPriceBid, baseFee, gasPriceL1FeedAddr, maxGas);
```

Comment:
This syntax is quite error prone. Solidity allows you to use named struct construction like this

ARM.Voter({
      blessVoteAddr: BLESS_VOTER_4,
      curseVoteAddr: CURSE_VOTER_4,
      curseUnvoteAddr: CURSE_UNVOTER_4,
      blessWeight: WEIGHT_40,
      curseWeight: WEIGHT_40
    });
    
    
 This way you can never swap args and accidentally assign the wrong values